Sources/ConsoleKit/Command/CommandGroup.swift

```diff
@@ -82,7 +82,9 @@ extension CommandGroup {
             guard let command = self.commands[name] else {
                 throw CommandError(
```

Comment:
I think `CommandError` should be an enum. Then we can have a case like `CommandError.notFound(name: String, available: [String])`.

Reply:
I guess that makes more sense as a separate PR

Reply:
Done in https://github.com/vapor/console-kit/pull/122